test/test_nn_quantized.py

```diff
@@ -1,26 +1,49 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
 import torch
 import torch.nn.quantized.functional as F
-
+import torch.nn.quantized as nnq
 import numpy as np
 from common_utils import TestCase, run_tests
 
-def _quantize(x, scale, zero_point, qmin=0, qmax=255):
-    """Quantizes a numpy array."""
-    qx = np.round(x / scale + zero_point)
-    qx = np.clip(qx, qmin, qmax).astype(np.uint8)
-    return qx
-
 class FunctionalAPITest(TestCase):
-    def test_functional_api(self):
+    def test_relu_api(self):
         X = torch.arange(-5, 5, dtype=torch.float)
         scale = 2.0
         zero_point = 1
-        Y = X.numpy().copy()
-        Y[Y < 0] = 0
-        qY = _quantize(Y, scale, zero_point)
         qX = torch.quantize_linear(X, scale=scale, zero_point=zero_point, dtype=torch.quint8)
+        # check it is the same as calling torch.ops.quantized.relu directly
+        qY = torch.ops.quantized.relu(qX)
         qY_hat = F.relu(qX)
-        np.testing.assert_equal(qY, qY_hat.int_repr())
+        self.assertEqual(qY, qY_hat)
+
+
+class ModuleAPITest(TestCase):
+    def test_linear_api(self):
+        """test API functionality for nn.quantized.linear"""
+        input_channels = 10
```

Comment:
Please, write more comprehensive tests.

Reply:
@zafartahirov this is API test, not functionality test, functionality test should be done in `test_quantized.py`

Reply:
I'll add a comment to the test file.